### PR TITLE
build towncrier in precommit

### PR DESCRIPTION
**Pull request by tardyp**
Looks like I messed up the commit, and forgot to include the makefile part

### CODE REVIEW

**Comment by tardyp** [comment]
Problem is that towncrier generates a relnote that does not pass the spell checks.

**Comment by sa2ajj** [comment]
(Side note: I could not possibly add `in progress` label at the time it says I did)

**Comment by sa2ajj** [comment]
(it's probably via waffle integration, removed)

File: master/docs/Makefile [hunk 1]
```diff
@@ -48,38 +48,41 @@ help:
 clean:
 	-rm -rf $(BUILDDIR)/*
 
-html: conf.py
+towncrier:
+	cd ../../; yes n | towncrier
```
Comment (sa2ajj) [inline master/docs/Makefile]: this look hackish: no way to provide `yes` to `towncrier` otherwise?

**Pull request merged**